seed/tests/test_portfoliomanager.py

```diff
@@ -29,7 +29,7 @@
 
 # override this decorator for more pressing conditions
 try:
-    pm_avail_check = requests.get('http://isthewallbuilt.us/api.json', timeout=5)
+    pm_avail_check = requests.get('http://isthewallbuilt.inbelievable.com/api.json', timeout=5)
```

Comment:
@Myoldmopar -- good?

Reply:
I get the expected response when querying that url, looks like it should be good for a while.  :+1: